charts/tempo-distributed/README.md.gotmpl

```diff
@@ -39,10 +39,10 @@ The command removes all the Kubernetes components associated with the chart and
 
 A major chart version change indicates that there is an incompatible breaking change needing manual actions.
 
-### from Chart versions < 1.11.0
+### from Chart versions < 1.12.0
```

Comment:
"Less than" sign is confusing here. AFAIU with recent changes it should be 
```suggestion
### from Chart versions < 1.13.1
```
Or 1.13.0 IDK.